virt/virt-4-10-release-notes.adoc

```diff
@@ -284,3 +284,9 @@ $ oc annotate --overwrite -n openshift-cnv hco kubevirt-hyperconverged 'networka
 
 * The web console does not display virtual machine templates that are deployed to a custom namespace. Only templates deployed to the default namespace display in the web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054650[*BZ#2054650*])
 ** As a workaround, avoid deploying templates to a custom namespace.
+
+* On a Single Node OpenShift (SNO) cluster, if a VMI has the `spec.evictionStrategy` field set to `LiveMigrate`, then you cannot migrate or evict the VMI or upgrade the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2073880[*BZ#2073880*])
```

Comment:
I am confused about why we are mentioning that migration fails when it is not supported on SNO in the first place. Here is a suggested rewrite (including a couple of minor wording changes):

```
* On a Single Node OpenShift (SNO) cluster, updating the cluster fails if a VMI has the `spec.evictionStrategy` field set to `LiveMigrate`. This is because migration is not supported on a single node cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2073880[*BZ#2073880*])
** There are two workaround options:
*** Remove the `spec.evictionStrategy` field from the VM declaration.
*** Manually stop the VM before you update {product-title}.
```

@stu-gott Thoughts on this rewrite?

Reply:
Thank you @ousleyp. Your rephrasing makes things more clear.